src/libs/maintaining-std.md

```diff
@@ -223,8 +223,13 @@ Features can be stabilized in a PR that replaces `#[unstable]` attributes with `
 
 You can find the right version to use in the `#[stable]` attribute by checking the [Forge].
 
+### When a `const` function is being stabilized
+
+Const functions can be stabilized in a PR that replaces `#[rustc_const_unstable]` attributes with `#[rustc_const_stable]` ones. The [Constant Evaluation WG] should be pinged for input on whether or not the `const`-ness is something we want to commit to. This is particularly important when the function internally depends on other unstable `const` functions through `#[allow_internal_unstable]` attributes.
```

Comment:
It's important that where `unsafe` and `const` is involved, e.g., for operations which are "unconst", that the const safety argument for the usage also be documented. That is, a `const fn` has additional determinism (e.g. run-time/compile-time results must correspond and the function's output only depends on its inputs...) restrictions that must be preserved, and those should be argued when `unsafe` is used.